build.sbt

```diff
@@ -0,0 +1,15 @@
+scalaVersion := "2.12.4"
+
+lazy val root = project
+  .in(file("."))
+  .settings(
+    sbtPlugin := true,
+    organization := "com.lightbend.akka",
```

Comment:
Maybe `com.lightbend.paradox`? (https://github.com/lightbend/paradox/blob/master/build.sbt#L22)

Reply:
Thanks, corrected this in the new commit https://github.com/lightbend/sbt-paradox-unidoc/pull/2/commits/f02eeff24b79ff9d9b34c3b1d7b8c34ff8cd24ce